frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -266,16 +266,22 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
   };
 
   const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
-    return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
-    });
+    const patch = [
+      {
+        op: 'add',
+        path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
```

Comment:
Out of curiousity, where does `'/metadata/labels/cluster.ocs.openshift.io~1openshift-storage'` come from? the `~1openshift-storage` seems a little strange to me.